virttest/utils_misc.py

```diff
@@ -2402,6 +2402,8 @@ def get_qemu_version(params):
             version['update'] = int(search_result.group(3))
         if "rhev" in str(line).lower():
             version['is_rhev'] = True
+        if '-ma' in str(line).lower():
+            version['is_ma'] = True
```

Comment:
https://github.com/avocado-framework/avocado-vt/blob/58cae3af968a4be113720a586a97b08bb3f017e6/virttest/utils_misc.py#L2338-L2341

Let us have the new key 'is_ma' in version init and in doc string.
Code looks good to me otherwise

Reply:
Thanks for the review. Updated.